tools/esdlcmd/esdlcmd_core.cpp

```diff
@@ -276,9 +276,9 @@ class Esdl2XSDCmd : public EsdlHelperConvertCmd
     {
         puts("Usage:");
         puts("esdl xsd sourcePath serviceName [options]\n" );
-        puts("\nsourcePath must be absolute path to the ESDL Definition file containing the" );
-        puts("EsdlService definition for the service you want to work with.\n" );
-        puts("serviceName EsdlService definition for the service you want to work with.\n" );
+        puts("\nsourcePath - Absolute path to the EXSDL Definition file (XML) containing the" );
```

Comment:
ECM is not XML, and certainly not the EXSDL required by this feature.
In 6.2.0 we'll add a feature to create the EXSDL if an ECM definition is provided.